.github/workflows/test.yml

```diff
@@ -32,9 +28,20 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: s-weigand/setup-conda@v1
```

Comment:
If we need conda, consider https://github.com/conda-incubator/setup-miniconda

Reply:
Thanks Bob! Let me try it now.

Reply:
The conda should be necessary as I checked that the `rdkit` is not a pure python package so that the package author said it is not supported to distribute with `PyPi`